testing/fuchsia/run_tests.py

```diff
@@ -51,11 +51,16 @@
 OUT_DIR = os.path.join(DIR_SRC_ROOT, 'out', VARIANT)
 
 
+class TestCase(NamedTuple):
+  package: str = None
```

Comment:
Needs to be `str | None`

Reply:
I removed the `= None`, it should not be None indeed.